azavea/ashlar-blueprint
# Build backend schema editor

## Overview

_Currently under development. Once ready, I'll remove the `wip` tag and request a review._

This PR fleshes out the backend portion of an Ashlar reference implementation: the schema editor.

### Checklist


### Demo

TK.

### Notes

TK.

## Testing Instructions

 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output

Closes https://github.com/azavea/ashlar-2018-fellowship/issues/5.
